lib/resources/kernel_module.rb

```diff
@@ -36,19 +68,51 @@ def loaded?
       !found.nil?
     end
 
-    def version
-      if inspec.os.redhat? || inspec.os.name == 'fedora'
-        modinfo_cmd = "/sbin/modinfo -F version #{@module}"
-      else
-        modinfo_cmd = "modinfo -F version #{@module}"
-      end
+    def disabled?
+      !modprobe_output.match(%r{^install\s+#{@module}\s+/(s?)bin/(true|false)}).nil?
+    end
 
-      cmd = inspec.command(modinfo_cmd)
+    def disabled_via_bin_true?
```

Comment:
So your thinking we just add bin_true and bin_false to the blacklisted? method and use a case statement?